about.html

```diff
@@ -94,7 +94,7 @@ <h3>Partner</h3>
         <div class="container-fluid footeres">
             <div class="row">
                 <div class="col">
-                  <img src="./images/creativecommons.png" alt="Creative Commons">
+                  <img src="./logos/ccBlack.png" alt="Creative Commons" class="mobileF"><img src="./logos/ccWhite.png" alt="Creative Commons" class="deskF"><div>creative</br>commons korea</div> 
                 </div>
                 <div class="col">
                   <p>2015 Creative Commons Korea</br>Some Rights Reserved</p>
```

Comment:
*[OPTIONAL]*

- Your footer is not well aligned compared to the original template. Please let's keep in mind that this project helps test how we can pay more attention to details as this is a very important skill that will make you standout in the job market.